searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java

```diff
@@ -81,8 +82,9 @@ public static void faceDetection (String file, String type) throws FaceDetection
             //Save the image already processed with the face detection
             Imgcodecs.imwrite(currentDir + "\\src\\main\\resources\\images\\" + finalFile, src);            //logger: info that everything it right
             log.info("It's only one person in the image");
+            status = true; //status of the face detection
             //result if the image it's a profile
-            result = "profile detected";//"";
+            result = "it's a person: " + status;//"";
```

Comment:
Please remove this //"";      or add an explanation of what it is for.

Reply:
already erase the "//"